cmake/std/atdm/utils/get_known_system_info.sh

```diff
@@ -150,6 +150,10 @@ fi
 
 # CTS1 systems
 if [[ $SNLSYSTEM == "cts1" ]] ; then
+  # Make cts1empire the default environment
+  systemNameTypeMatchedList+=(cts1empire)
+  systemNameTypeMatchedListHostNames[cts1empire]=$SNLCLUSTER
```

Comment:
I think we also need to add 'cts1empire' to the above list, no?  (We should refactor this script so that is not needed but for now it is.)

Reply:
That is, we need to add 'cts1empire' right above 'cts1' here:

https://github.com/trilinos/Trilinos/blob/ad04133a954c6e4f69502a3a868c25dee85bd317/cmake/std/atdm/utils/get_known_system_info.sh#L68